Makefile

```
@@ -0,0 +1,14 @@
+# Makefile, redirecting to "make.py"
+
+# Sphinx options, can be set from the command line
+OPTS =
+
+# This runs for `make` without any arguments
+all:
+	python3 make.py
+
+html:
+	python3 make.py html $(OPTS)
+
+check-links:
```

Pretty sure its `make linkcheck`, as it is just `make buildername` and `linkcheck` is the builder.

```suggestion
linkcheck:
```